components/PostSpecific/PostSpecific.tsx

```diff
@@ -0,0 +1,55 @@
+import SearchTag from 'components/SearchTag/SearchTag';
+import styles from './PostSpecific.module.css';
+import Critique from 'components/Critique/critique';
+
+export default function PostSpecific({ post }: { post: any }) {
+    const { author, createdDate, title, desc, src, likes, critiques } = post;
+    const searchTags = [
+        { name: 'Painting', color: '#ffa9a9' },
+        { name: 'Illustration', color: '#efb190' },
+        { name: 'Sculpture', color: '#9bd634' },
+    ];
+
+    return (
+        <div className={styles.postContainer}>
+            <div className={styles.postHeader}>
+                {`Posted by ${author} ${createdDate} days ago`}
+            </div>
+            <div className={styles.postTitle}>{title}</div>
+            <div className={styles.postDesc}>{desc}</div>
+            <div className={styles.postContent}>
+                <img src={src} className={styles.image} />
+            </div>
+            <div className={styles.searchTagContainer}>
+                {searchTags.map((tagObject) => (
+                    <div>
+                        <SearchTag color={tagObject['color']}>
```

Comment:
not anything to note, but i think becca wanted to stick with dot notation instead of brackets

either works though :)